Kiosk/App/Networking/ArtsyAPI.swift

```diff
@@ -28,6 +28,7 @@ enum ArtsyAPI {
     
     case MyBiddersForAuction(auctionID: String)
     case MyBidPositionsForAuctionArtwork(auctionID: String, artworkID: String)
+    case MyBidPosition(id: String)
```

Comment:
Could we use a more descriptive variable name? Something like `saleArtworkID` or whatever this relates to :)